src/runtime_src/core/common/query_requests.h

```diff
@@ -2991,6 +2992,15 @@ struct hwmon_sdm_active_msp_ver : request
   get(const device*) const = 0;
 };
 
+struct hwmon_sdm_target_msp_ver : request
```

Comment:
Please add a few lines comment about what this request is.

Reply:
Hi @stsoe Thank you for your review. I have taken care of your suggestions. Thank you.